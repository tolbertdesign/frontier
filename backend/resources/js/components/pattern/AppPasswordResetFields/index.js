import AppPasswordResetFields from './AppPasswordResetFields.vue'

export default AppPasswordResetFields
