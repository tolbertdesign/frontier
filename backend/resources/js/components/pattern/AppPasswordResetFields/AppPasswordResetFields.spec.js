import { shallowMount, createLocalVue } from '@vue/test-utils'
import AppPasswordResetFields from './AppPasswordResetFields.vue'

const localVue = createLocalVue()

const emailAddressLabel = 'email address label'
const passwordLabel = 'password label'
const passwordConfirmLabel = 'password confirm label'
const error = 'error'

describe('AppPasswordResetFields', () => {
  it('Snapshop test for AppPasswordResetFields', () => {
    const wrapper = shallowMount(AppPasswordResetFields, {
      localVue,
      propsData: {
        emailAddressLabel: emailAddressLabel,
        passwordLabel: passwordLabel,
        passwordConfirmLabel: passwordConfirmLabel,
        error: error,
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
