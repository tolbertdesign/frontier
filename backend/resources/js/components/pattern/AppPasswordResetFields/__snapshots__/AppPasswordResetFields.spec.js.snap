// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppPasswordResetFields Snapshop test for AppPasswordResetFields 1`] = `
<div>
  <div class="form-group {"><label for="email" class="sr-only">

    </label> <input type="email" name="email" required="required" class="is-invalid form-control mw-250px mx-auto"></div>
  <div class="form-group"><label for="email" class="sr-only">

    </label> <input type="password" name="password" required="required" class="is-invalid form-control mw-250px mx-auto"></div>
  <div class="form-group"><label for="email-confirm" class="sr-only">

    </label> <input type="password" name="password_confirmation" required="required" class="is-invalid form-control mw-250px mx-auto"></div>
</div>
`;
